### PR TITLE
Add a Mobile menu using javascript

### PR DESCRIPTION
**Pull request by zunairkhan811**
The mobile menu feature has been made according to the following requirements:
-When the user clicks (or taps) the hamburger button, the mobile menu appears.
-When the user clicks (or taps) the close (X) button, the mobile menu disappears.
-When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
-When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,17 +22,15 @@
...
       <nav class="nav-bar">
         <div class="left-navbar">
-          <a href="#" class="logo">Muhammad Zunair khan</a>
-          <div class="wrap">
-            <a href="#">
-              <div class="menu"></div>
-              <div class="menu"></div>
-              <div class="menu"></div>
-            </a>
+          <a id="logo" href="#" class="logo">Muhammad Zunair khan</a>
```
Comment (Ismailco) [inline index.html:25]: - [x] When clicking on any link on the mobile menu the menu should disappear, kindly make sure to implement this so the website behaves as expected

File: index.html [hunk 2]
```diff
@@ -22,17 +22,15 @@
...
+          <div id="wrap" class="wrap" onclick="onClickMenu()">
```
Comment (Ismailco) [inline index.html]: - [x] The mobile menu should take the whole height of the page when open, kindly consider implementing this so your site looks identical to the Figma design
- [x] Also kindly make sure the font wight and the close button color is similar to the Figma design

|Your website|Figma design|
|-|-|
|Screenshot_20230130_210351|Screenshot_20230130_210552|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @zunairkhan811, @anita00001, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Team, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡
 
✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**